src/components/SupportRequests.tsx

```diff
@@ -0,0 +1,147 @@
+/* global fetch */
+import React from 'react';
+import DataTable, { TableColumn } from 'react-data-table-component';
+import { SupportRequestType } from '../types/supportRequest';
+import { DateTime } from 'luxon';
+
+export interface Team {
+  id: number;
+  createdAt: string;
+  updatedAt: Date;
+  movedToInProgressAt: Date;
+  slackId: string;
+  name: string;
+  supportName: string;
+  status: string;
+  type: SupportRequestType;
+  syncHash: string;
+}
+
+interface SupportRequestsProps {
+  secret: string;
+}
+
+let columns: TableColumn[] = [];
+
+function toTitleCase(text: string): string {
+  return text.replace(/([A-Z])/g, ' $1').replace(/^./, (firstChar) => firstChar.toUpperCase());
+}
+
+const ExpandableRow: React.FC<{ data?: Team }> = (props) => (
+  <div className="py-2 px-3">
+    <p className="mb-1">Description</p>
+    <p style={{ fontSize: 12 }}>{props.data?.createdAt ?? '¯\\_(ツ)_/¯'}</p>
+  </div>
+);
+
+export const SupportRequests: React.FC<SupportRequestsProps> = ({ secret }) => {
+  const [error, setError] = React.useState(false);
+  const [teams, setTeams] = React.useState<Team[]>([]);
+
+  const timeoutRef = React.useRef<NodeJS.Timeout>(null);
+
+  const fetchValues = async (): Promise<void> => {
+    const res = await fetch('/api/supportRequest/getAll', { headers: { authorization: secret } });
+
+    if (!res.ok) {
+      setError(true);
+      return;
+    }
+
+    const teamData: Team[] = await res.json();
+    const keys = Object.keys(teamData[0] ?? {}).filter(
+      (key) => !['id', 'createdAt', 'updatedAt', 'movedToInProgressAt', 'slackId', 'name', 'supportName', 'status', 'type', 'syncHash'].includes(key),
+    );
+    teamData.forEach((team: Team) => {
+      Object.keys(team).forEach((key) => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        var val = (team as any)[key as keyof Team];
+        console.log(key);
+        console.log(val);
+        if (key == 'createdAt') {
+          console.log('HELLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLo');
```

Comment:
Make sure to remove the `console.log` statements since we don't want that logged to anyone in production